src/main/java/org/jvnet/hudson/test/junit/jupiter/InboundAgentExtension.java

```diff
@@ -630,7 +630,7 @@ private static void waitForAgentOnline(JenkinsRule r, String name, Map<String, L
             throw new AssertionError("no such agent: " + name);
         }
         if (!(node instanceof Slave)) {
-            throw new AssertionError("agent is not a Slave: " + name);
+            throw new AssertionError("agent is not a Agent: " + name);
```

Comment:
Capitalization mistake in the original string.
```suggestion
            throw new AssertionError("agent is not an agent: " + name);
```